condition/asset.go

```diff
@@ -0,0 +1,34 @@
+package condition
+
+import (
+	"net"
+
+	"github.com/google/uuid"
+	"github.com/metal-toolbox/rivets/types"
+)
+
+// Asset holds attributes of a server retrieved from the inventory store.
+//
+// nolint:govet // fieldalignment struct is easier to read in the current format
+type Asset struct {
+	ID uuid.UUID
+
+	// Device BMC attributes
+	BmcAddress  net.IP
+	BmcUsername string
+	BmcPassword string
+
+	// Inventory status attribute
+	State string
```

Comment:
Where does this come from?

Reply:
This field isn't in use and should have been removed earlier.